src/Screens/Main/SearchMentor.tsx

```diff
@@ -37,6 +39,10 @@ export default ({ navigation }: Props) => {
   const dispatch = useDispatch<redux.Dispatch<actions.Action>>();
 
   const allSkills = useSelector(mentorState.getSkillList);
+
+  const { hideInactiveMentors } = useSelector(
```

Comment:
Here too, add filterSkill and remove selectedSkills from ln 46

Reply:
Yesss